signal0ne/src/extension.ts

```diff
@@ -149,4 +154,8 @@ export async function activate(context: vscode.ExtensionContext) {
   });
 }
 
-export function deactivate() {}
+export function deactivate() {
+  // if (issueDetailsSidePanel) {
+  //   issueDetailsSidePanel.dispose();
+  // }
```

Comment:
this is the disposal that didn't work? maybe we need onDispose handler : https://code.visualstudio.com/api/extension-guides/webview#lifecycle